pkg/netlinkwrapper/netlink.go

```diff
@@ -69,6 +75,42 @@ type NetLink interface {
 type netLink struct {
 }
 
+const (
+	// Environment variable to configure netlink max retry attempts
+	envNetlinkMaxRetries = "AWS_VPC_K8S_CNI_NETLINK_MAX_RETRIES"
+	defaultMaxAttempts   = 5
+)
+
+// getMaxAttempts returns the configured maximum retry attempts for netlink operations
+func getMaxAttempts() int {
+	maxAttempts, _, _ := utils.GetIntFromStringEnvVar(envNetlinkMaxRetries, defaultMaxAttempts)
```

Comment:
I believe CNI won't be able to access this env variable since it's running as a binary on the host. Might need to pass this via conf file or the GRPC call